api/v1/deployment/repository.go

```diff
@@ -144,11 +147,13 @@ func (repo *Repository) FindBy(criterias map[string]interface{}) (interface{}, e
 	return deps, err
 }
 
-// FindActivesBy fetch a collection of deployments matching each criteria
-func (repo *Repository) FindActivesBy(criterias map[string]interface{}) ([]*v1.Deployment, error) {
+// FindActivesBy fetch a collection of deployments matching each criteria on a given domain and name apps
+func (repo *Repository) FindActivesBy(domain string, name string, criterias map[string]interface{}) ([]*v1.Deployment, error) {
 	var deps []*v1.Deployment
-
-	err := repo.db.Preload("Environment").Preload("Application").Model(v1.Deployment{}).Where("undeployed_at IS NULL").Find(&deps, criterias).Error
+	err := repo.db.Table("deployments").
```

Comment:
Removal of *preloads* will prevents the hateoas link generation (i.e. navigation from a deployment to its release), no?